package.json

```diff
@@ -61,6 +61,13 @@
       "eslint --fix --ext js,jsx,ts,tsx"
     ]
   },
+  "pnpm": {
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "react": "17"
```

Comment:
For information, this is due to `use-media` which requires react 16 in peer deps, see https://github.com/streamich/use-media/issues/183

Reply:
As use-media does not seems to be maintained anymore, can we plan to fork it or have our own brew internally ?

Reply:
We can create a new package `@scaleway/use-media` but it will reside in [scaleway-lib](https://github.com/scaleway/scaleway-lib), so not a real "fork"